### ISSUE TITLE
Platform table filter - lack of aria-label for listbox option

### ISSUE THREAD

**Issue by esporysz** [SAP/fundamental-ngx#12780]
#### Is this a bug, enhancement, or feature request?

bug

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

angular: 18.2.4
fundamental: 0.53.1-rc.23

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;

1. Crete some platform table with  filtering. It may be copy of [example from documentation](https://sap.github.io/fundamental-ngx/#/platform/table/preserved-state).
2. Listbox options for Filter By do not have `aria-label`

**Comment by droshev** [comment]
@dpavlenishvili will your table settings dialog cover that?

**Comment by mikerodonnell89** [comment]
Closing this issue, options are read by the screenreader for both the filter dialog and the filter by options in the combined dialog
